C/CGAL/build_tarballs.jl

```diff
@@ -41,19 +36,18 @@ install_license CGAL-*/LICENSE*
 platforms = expand_cxxstring_abis(supported_platforms())
 
 # The products that we will ensure are always built
-products = [
-    LibraryProduct("libCGAL", :libCGAL),
-    LibraryProduct("libCGAL_Core", :libCGAL_Core),
-    LibraryProduct("libCGAL_ImageIO", :libCGAL_ImageIO),
-]
+# CGAL is, as of 5.0, a header-only library, removing support for lib
+# compilation in 5.3
+products = Product[]
```

Comment:
Header-only?  Then, if the resulting tarball is platforms independent you want to build for
```julia
platforms = [AnyPlatform()]
```

Reply:
Interesting, this is new to me. I'll update accordingly ASAP.